homeworkLecturesW11D05/src/ba/bitcamp/homeworkLecturesW11D5/Client.java

```diff
@@ -0,0 +1,60 @@
+package ba.bitcamp.homeworkLecturesW11D5;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.Socket;
+import java.net.UnknownHostException;
+
+import org.codehaus.jackson.map.ObjectMapper;
+
+/**
+ * 
+ * @author Gordan
+ *
+ *         This class represents one client connected to local server. Client
+ *         receives one JSON file from server and saves it. After that client
+ *         constructs object from that file.
+ */
+public class Client {
+
+	public static void main(String[] args) {
```

Comment:
Instead of putting all the code into main method, class should have some structure:

- Socket should be private member
- There should be a method that inits the socket using host/port
- There should be a method that parses the message
- There should be a method that handles message sending